src/dao/pages/DAOProposals/Forms/DAOTokenTransferDetailsForm.tsx

```diff
@@ -132,40 +158,24 @@ export function DAOTokenTransferDetailsForm() {
         isInvalid={Boolean(errors?.recipientAccountId)}
         errorMessage={errors?.recipientAccountId && errors?.recipientAccountId?.message}
       />
-      <FormDropdown
-        label="Token Type"
-        placeholder="Select a token type"
-        data={[
-          { label: "Fungible Token", value: TokenType.FungibleCommon.toString() },
-          { label: "NFT", value: TokenType.NonFungibleUnique.toString() },
-        ]}
-        isInvalid={Boolean(errors?.tokenType)}
-        errorMessage={errors.tokenType && errors.tokenType.message}
-        register={register("tokenType", {
-          required: { value: true, message: "A token type is required to be selected to transfer" },
-          onChange: (e: ChangeEvent<HTMLSelectElement>) => {
-            setValue("tokenType", e.target.value), setValue("tokenId", "");
-          },
-        })}
-      />
-      {tokenType === TokenType.FungibleCommon.toString() && (
+      {tabIndex === DepositTokenModalTabs.Fungible && (
         <FormTokenInput
           amountFormId="amount"
           tokenFormId="tokenId"
           assetListAccountId={safeAccountId}
           balanceAccountId={safeAccountId}
-          initialSelectedTokenId={state?.tokenId}
+          initialSelectedTokenId={getValues().tokenId}
```

Comment:
You should be able to use the `tokenId` variable on line 72 instead of invoking the `getValues()` function an addtional time.

Reply:
Done.